en/sim_sih/index.md

```diff
@@ -164,6 +164,45 @@ export PX4_HOME_ALT=28.5
 make px4_sitl sihsim_quadx
 ```
 
+
+### Adding new airframes
+
+[Adding a new airframe](../dev_airframes/adding_a_new_frame.md) for use in SIH simulation is much the same as for other use cases.
+You still need to configure your vehicle type and [geometry](../config/actuators.md) (`CA_` parameters) and start any other defaults for that vehicle type.
```

Comment:
```suggestion
You still need to configure your vehicle type and [geometry](../config/actuators.md) (`CA_` parameters) and start any other defaults for that specific vehicle.
```

The vehicle type in PX4 generally multicopter, VTOL, fixed wing, rover, ... related things should be covered in e.g. https://github.com/PX4/PX4-Autopilot/blob/57fdda597be4e14e3f493f6b55ef550bbb9bdccd/ROMFS/px4fmu_common/init.d/rc.mc_defaults vehicle type I'd only use to refer to that.